### DISCUSSION TITLE
Any plan for higher resolution?

### DISCUSSION THREAD

**Discussion by everdancer** [jasonjgardner/jg-rtx#62] [Feedback]
Hi @jasonjgardner , current textures are so good! Just wonder if you have any plan to provide higher resolution >= 512x? When I use it under 4K, details are not perfect.

**Comment by jasonjgardner** [comment]
Unfortunately, I'll likely have to keep the pack at 256x. I started this project at 512x, but quickly hit Minecraft's 100mb pack size limit, and had to reduce the resolution to continue development. The limit is now around 300mb, from what I've heard. JG RTX is currently 250mb. I think I'll be pushing that limit once the remaining textures have been completed.

Check out Kingdom of Wallace for a 512x Bedrock RTX pack. It's only available through Patreon, but it's very high-quality: https://www.patreon.com/wallace71DNK

**Reply by everdancer** [reply]
Thanks! I will look at Kingdom pack.
